### DISCUSSION TITLE
Authentication methods available for the app

### DISCUSSION THREAD

**Discussion by DBH1986** [github-copilot-community/copilot-metrics-viewer#97] [Q&A]
Hello, I have this app deployed in Azure and I would like to add authentication so only my organization can access the App.  I set it up to use our MS identity provider

![image](https://github.com/user-attachments/assets/427658c1-28e4-4acb-930d-fa5b5af1d9a2)

Then set the redirect uri back to the app

![image](https://github.com/user-attachments/assets/66fe5a00-2b88-4169-a5a8-95eb783dd380)

It is hitting my companies identity provider but I am getting `AADSTS900971: No reply address provided.` It looks like I may have to configure something in the app itself to make this redirect uri, but on a quick code search nothing stood out to me. Is identity provider auth available for this application, and if not what authentication could I enable?

Thanks!

**Comment by karpikpl** [comment]
I know it's been a long time, but this seems something Azure Support can help with, the "EasyAuth" feature of Azure that you're using here is not part of the copilot metrics viewer application
